src/core/classes/items-management/list-items-manager.ts

```diff
@@ -196,8 +200,32 @@ export class CoreListItemsManager<
         super.updateSelectedItem(route);
 
         const selectDefault = CoreScreen.isTablet && this.selectedItem === null && this.splitView && !this.splitView.isNested;
+        const selectedItem = selectDefault ? this.getDefaultItem() : this.selectedItem;
+        this.select(selectedItem);
+    }
```

Comment:
```suggestion
        this.select(selectDefault ? this.getDefaultItem() : this.selectedItem);
    }
```

I think it's better to leave this as this for a cleaner diff (it's probably changed because we had a different approach before).